test/file-operations.js

```diff
@@ -76,24 +80,27 @@ describe('isOwner', function() {
       process.getuid = noop;
     }
 
-    getuidSpy = expect.spyOn(process, 'getuid').andReturn(ownerStat.uid);
-    geteuidSpy = expect.spyOn(process, 'geteuid').andReturn(ownerStat.uid);
+    getuidSpy = sinon.stub(process, 'getuid').returns(ownerStat.uid);
+    geteuidSpy = sinon.stub(process, 'geteuid').returns(ownerStat.uid);
 
     done();
   });
 
   afterEach(function(done) {
-    expect.restoreSpies();
+    process.geteuid = geteuidSpy;
+    process.getuid = getuidSpy;
 
-    if (process.geteuid === noop) {
-      delete process.geteuid;
+    if (typeof process.geteuid !== 'function') {
+      process.geteuid = noop;
     }
 
     // Windows :(
-    if (process.getuid === noop) {
-      delete process.getuid;
+    if (typeof process.getuid !== 'function') {
+      process.getuid = noop;
     }
 
+    sinon.restore();
+
```

Comment:
I don't understand this. Flagging to look at myself.

Reply:
It's my mistake. I've corrected this part and added an explanation comment.

Reply:
This looks good to me now.